src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -1207,6 +1207,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\Marshal.Unix.cs" Condition="'$(TargetsCoreRT)' != 'true'" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Security\SecureString.Unix.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Threading\Thread.Unix.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Threading\TimerQueue.Unix.cs" Condition="'$(FeatureNativeTimer)' != 'true'" />
```

Comment:
We have `FeaturePortableThreadPool`. Would `FeaturePortableTimer` be better name for this to follow the naming pattern?

Reply:
yeah, but where do I set the property in corert?

Reply:
https://github.com/dotnet/corert/blob/master/src/System.Private.CoreLib/src/System.Private.CoreLib.csproj#L46